config/openvpn-client-export/openvpn-client-export.inc

```diff
@@ -138,7 +138,15 @@ function openvpn_client_export_validate_config($srvid, $usrid, $crtid) {
 	if (!$server_cert) {
 		$input_errors[] = "Could not locate server certificate.";
 	} else {
+<<<<<<< Updated upstream
 		$server_ca = lookup_ca($server_cert['caref']);
+=======
+		/*
+			Replaced lookup_ca($server_cert['caref']) with ca_chain($server_cert). Note that ca_chain returns a string of Base64 encoded certificates, not an array. 
+			Thus, all references to base64_decode($server_ca['crt']) have been replaced with simply $server_ca
+		*/
+		$server_ca = ca_chain($server_cert); 
+>>>>>>> Stashed changes
```

Comment:
This was incorrect in this commit. I tried to remove it from the chain, but failed.

The code is correct in the final commit: 06ec0438662c73a2d70b615a9c40f5c0d30b3bd1.